emojivoto-web/cmd/server.go

```diff
@@ -23,6 +28,19 @@ func main() {
 		log.Fatalf("WEB_PORT (currently [%s]) EMOJISVC_HOST (currently [%s]) and VOTINGSVC_HOST (currently [%s]) INDEX_BUNDLE (currently [%s]) environment variables must me set.", webPort, emojisvcHost, votingsvcHost, indexBundle)
 	}
 
+	oce, err := ocagent.NewExporter(
+		ocagent.WithInsecure(),
```

Comment:
What does this control? Is this signaling that the grpc client shouldn't use tls?

Is there a pattern for deploying the OC agent securely in kubernetes that we may have to support?

Reply:
I believe so.  Let me investigate further.

Reply:
I'm not able to find any patterns or best practices.  If anyone is even using this in non-demo environments, I can't find anyone talking about it.

There is an undocumented feature where the agent and collector can serve over TLS: https://github.com/census-instrumentation/opencensus-service/pull/415